frontend/src/components/visual-editor/form/inputs/message/QuickRepliesInput.tsx

```diff
@@ -65,9 +61,18 @@ const QuickRepliesInput: FC<QuickRepliesInput> = ({
   };
 
   useEffect(() => {
-    onChange(quickReplies.map(({ value }) => value));
+    if (
+      quickReplies.length === 0 ||
+      (quickReplies.length === 1 &&
+      quickReplies[0]?.value?.title?.trim() === "" &&
+      quickReplies[0]?.value?.payload?.trim() === "")
+    ) {
+      onChange([]);
+    } else {
+      onChange(quickReplies.map(({ value }) => value));
+    }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [quickReplies]);
+  }, [quickReplies,onChange]);
 
```

Comment:
You can revert this block since it's not the source of the problem,

Reply:
Thanks for all the suggestions fixed the requested changes by you